components/Canvas.js

```diff
@@ -3,67 +3,120 @@ import styled from "styled-components";
 
 function lineWidthRandomizer() {
   const randomNumber = Math.random();
-  if (randomNumber <= 0.4) {
+  if (randomNumber <= 0.7) {
     return 2;
-  } else if (randomNumber <= 0.8) {
+  } else if (randomNumber <= 0.9) {
     return 8;
   } else {
     return 4;
   }
 }
 
-let firstLineWidth = lineWidthRandomizer();
-let secondLineWidth = lineWidthRandomizer();
-let thirdLineWidth = lineWidthRandomizer();
-let fourthLineWidth = lineWidthRandomizer();
-
-export default function CanvasOne() {
+export default function Canvas() {
   const canvasRef = useRef(null);
-  const [backgroundColor, setBackgroundColor] = useState(148);
+
+  const [color, setColor] = useState(148);
+  const [rotation, setRotation] = useState(45);
+  const [outline, setOutline] = useState(0);
+  const [variation, setVariation] = useState([
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+    lineWidthRandomizer(),
+  ]);
+
   useEffect(() => {
     const canvas = canvasRef.current;
     const context = canvas.getContext("2d");
+
     // Beginn drawing statements
-    context.fillStyle = `hsl(${backgroundColor}, 60%, 95%)`;
+    context.fillStyle = `hsl(${color}, 60%, 95%)`;
     context.fillRect(0, 0, canvas.width, canvas.height);
 
-    context.fillStyle = "hsl(148, 100%, 50%)";
+    context.fillStyle = `hsl(${color}, 100%, 50%)`;
     context.translate(150, 150);
-    context.rotate((0 * Math.PI) / 180);
+    context.rotate((rotation * Math.PI) / 180);
     context.fillRect(-40, -40, 80, 80);
 
-    context.lineWidth = firstLineWidth;
-
-    context.strokeStyle = `hsl(${backgroundColor}, 60%, 95%)`;
+    context.lineWidth = variation[0];
+    context.strokeStyle = `hsl(${color}, 60%, 95%)`;
     context.strokeRect(-22.5, -22.5, 45, 45);
 
-    context.strokeStyle = "hsl(148, 100%, 50%)";
-    context.lineWidth = secondLineWidth;
-    context.strokeRect(-60, -60, 120, 120);
+    context.strokeStyle = `hsl(${color}, 100%, 50%)`;
+
+    for (let i = 0; i <= outline - 1; i++) {
+      let x = -60 - 20 * i;
+      let y = -60 - 20 * i;
+      let width = 120 + 40 * i;
+      let height = 120 + 40 * i;
+      context.lineWidth = variation[i];
+      context.strokeRect(x, y, width, height);
+    }
```

Comment:
this looks a bit too much like copied from the internet. Which is okay for complicated function but make sure to make it "yours" so you are able to understand it even in 2 years :)
I can't think of a good name for i :D
```suggestion
    for (let i = 0; i <= outline - 1; i++) {
      let coordinateOnXAxis = -60 - 20 * i;
      let coordinateOnYAxis = -60 - 20 * i;
      let widthOfRectangle = 120 + 40 * i;
      let heightOfRectangle = 120 + 40 * i;
      context.lineWidth = variation[i];
      context.strokeRect(x, y, width, height);
    }
```

Reply:
Hi Merle, 

I would just like to point out that I didn't copy the code or the artwork.

The artwork is very simple and not very sophisticated, I can do it without copying anything. ;)

The technique to draw several elements in canvas with one loop is a technique I learned in canvas tutorials. Also using the variable i to change the coordinates and the size of the rectangles on each loop is a technique I already knew before our bootcamp.

If I should copy things, which is not planned, then I will discuss this with you couches beforehand and mark it accordingly in my code as agreed.

But you are of course right, that "coordinateOnXAxis" is more self-explanatory than just "x".  Will be changed!!! ;)

Best regards and thanks for your review :)

Christian :)